deploy-manage/tools.md

```diff
@@ -5,28 +5,48 @@ mapped_pages:
 
 # Backup, high availability, and resilience tools [high-availability]
 
-Your data is important to you. Keeping it safe and available is important to Elastic. Sometimes your cluster may experience hardware failure or a power loss. To help you plan for this, {{es}} offers a number of features to achieve high availability despite failures. Depending on your deployment type, you might need to provision servers in different zones or configure external repositories to meet your organization’s availability needs.
+Elasticsearch provides comprehensive tools to safeguard data, ensure continuous availability, and maintain resilience. These tools are designed to support disaster recovery strategies, enabling businesses to protect critical information and minimize downtime, and maintain high availability in case of unexpected failures.
 
-* **[Design for resilience](production-guidance/availability-and-resilience.md)**
+For strategies to design resilient clusters, see **[Availability and resilience](production-guidance/availability-and-resilience.md)**.
 
-    Distributed systems like Elasticsearch are designed to keep working even if some of their components have failed. An Elasticsearch cluster can continue operating normally if some of its nodes are unavailable or disconnected, as long as there are enough well-connected nodes to take over the unavailable node’s responsibilities.
+::::{note} 
+Snapshot and restore, as well as cross-cluster replication, are not currently available in serverless deployments, but they are planned for future releases. For more information, see **[LINK-TODO]**.
+::::
```

Comment:
- prefer `Elastic Cloud Serverless` when you have the room to spell it out
- Elastic Cloud Serverless uses `projects`, not `deployments`.
- ECS also doesn't have "releases" per se (it's versionless), so we would just say "future"

provided an anchor to link down right to the feature plans for serverless

```suggestion
::::{note} 
The snapshot and restore and cross-cluster replication features are currently not available for Elastic Cloud Serverless projects. These features will be introduced in the future. For more information, refer to [Serverless differences](/deploy-manage/deploy/elastic-cloud/differences-from-other-elasticsearch-offerings.md#elasticsearch-differences-serverless-feature-categories).
::::
```